graylog2-web-interface/src/views/components/aggregationwizard/aggregationElements/GroupByElement.test.ts

```diff
@@ -107,4 +107,38 @@ describe('GroupByElement', () => {
       });
     });
   });
+
+  describe('remove grouping', () => {
+    const { removeElement } = GroupByElement;
+    const values = { groupBy: { columnRollup: true, groupings: [] } } as WidgetConfigFormValues;
+
+    it('should remove form values from a group by', () => {
```

Comment:
I prefer the wording you used for the `MetricElement` test, e.g. "should remove a grouping from the form".

Reply:
Same for the nexts test cases.